src/models/dfspTransaction.model.ts

```diff
@@ -422,6 +440,7 @@ export class DFSPTransactionModel
         // This requires user input on the PISP side, so this number should be something reasonable, like 1 minute or so
         .wait(this.config.transactionRequestVerificationTimeoutSeconds * 1000)
     } catch (error) {
+      console.log(error)
```

Comment:
any reason for keeping this in?

Reply:
done here and throughout. this was just my lazy debugging for tests.